core/src/main/java/org/bitcoinj/core/PeerGroup.java

```diff
@@ -1380,13 +1476,13 @@ public void setConnectTimeoutMillis(int connectTimeoutMillis) {
      *
      * @param listener a listener for chain download events, may not be null
      */
-    public void startBlockChainDownload(PeerDataEventListener listener) {
+    public void startBlockChainDownload(ChainDownloadStartedEventListener listener) {
```

Comment:
Do you want me to try fixing the compatibility break, or just charge forwards and rip the entire event out? Or fix the break and mark it deprecated?